homeassistant/components/websocket_api/commands.py

```diff
@@ -78,33 +78,43 @@ def async_register_commands(
 ) -> None:
     """Register commands."""
     async_reg(hass, handle_call_service)
+    async_reg(hass, handle_connection_info)
     async_reg(hass, handle_entity_source)
     async_reg(hass, handle_execute_script)
     async_reg(hass, handle_fire_event)
     async_reg(hass, handle_get_config)
     async_reg(hass, handle_get_services)
     async_reg(hass, handle_get_states)
-    async_reg(hass, handle_manifest_get)
+    async_reg(hass, handle_integration_descriptions)
     async_reg(hass, handle_integration_setup_info)
+    async_reg(hass, handle_manifest_get)
     async_reg(hass, handle_manifest_list)
     async_reg(hass, handle_ping)
     async_reg(hass, handle_render_template)
     async_reg(hass, handle_subscribe_bootstrap_integrations)
+    async_reg(hass, handle_subscribe_entities)
     async_reg(hass, handle_subscribe_events)
     async_reg(hass, handle_subscribe_trigger)
+    async_reg(hass, handle_supported_features)
     async_reg(hass, handle_test_condition)
     async_reg(hass, handle_unsubscribe_events)
     async_reg(hass, handle_validate_config)
-    async_reg(hass, handle_subscribe_entities)
-    async_reg(hass, handle_supported_features)
-    async_reg(hass, handle_integration_descriptions)
```

Comment:
aside from the one addition of `handle_connection_info`, these changes are just alphabetizing